node/node.go

```diff
@@ -579,13 +579,13 @@ func (n *Node) ProxyApp() proxy.AppConns {
 	return n.proxyApp
 }
 
-func (n *Node) makeNodeInfo(pubKey crypto.PubKey) p2p.NodeInfo {
+func (n *Node) makeNodeInfo(nodeID p2p.ID) p2p.NodeInfo {
```

Comment:
`id`

Reply:
there will some confusion, what ID we need to use. nodeID explicitly defines, which ID need to send in argument.